bin/replace-secrets.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Replaces redacted secret environment variables in an app
+ * directory's `redacted.env` and outputs `.env` with the decrypted values of
+ * the secrets.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const YAML = require('yaml');
+const crypto = require('crypto');
```

Comment:
nit: Sort requires. (We should probably enable [this](https://github.com/ampproject/amphtml/blob/d5b77757ede4937436ef1270622c94e8c7248f66/.eslintrc#L185) `amphtml` lint rule with auto-fix at a later point.)

Reply:
Done.